dashboard/db/migrate/20170202090015_add_school_info_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddSchoolInfoToUser < ActiveRecord::Migration[5.0]
```

Comment:
Note: this (as well as the next migration) shows up as a diff against staging, but should actually be applied when https://github.com/code-dot-org/code-dot-org/pull/13074 is merged)